Formula/percol.rb

```diff
@@ -0,0 +1,37 @@
+class Percol < Formula
+  desc "Interactive grep tool"
+  homepage "https://github.com/mooz/percol"
+  url "https://github.com/mooz/percol/archive/v0.2.1.tar.gz"
+  sha256 "75056ba1fe190ae4c728e68df963c0e7d19bfe5a85649e51ae4193d4011042f9"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  resource "six" do
+    url "https://pypi.python.org/packages/source/s/six/six-1.10.0.tar.gz"
```

Comment:
@zmwangx https://github.com/tdsmith/homebrew-pypi-poet is generally the way of generating these blocks and @tdsmith has recently switched over the URLs to the pythonhosted.org ones.